src/api/notification/controllers/notification.ts

```diff
@@ -5,19 +5,58 @@
 import { factories } from '@strapi/strapi'
 
 const MODULE_ID = 'api::notification.notification'
+const GLOBAL_MODULE_ID = 'api::notifications-consumer.notifications-consumer'
+const SINGLETON_ID = 1
 
 export default factories.createCoreController(MODULE_ID, ({ strapi }) => {
   return {
     async getNotificationList(context) {
       const account = context.params.account
 
-      return strapi.service(MODULE_ID).getNotificationList(account, false)
+      return strapi.service(MODULE_ID).getNotificationList(account)
     },
 
-    async getPushNotifications(context) {
-      const account = context.params.account
+    async getPushNotifications() {
+      const global = await strapi.entityService.findOne(GLOBAL_MODULE_ID, SINGLETON_ID, {
```

Comment:
ubernit: as you don't use global, and is not a pretty name for a var, I would just destruct it right away

Reply:
Renamed to `notificationsConsumer`, it cannot be destructed right here, because it can be null